Sources/Tabs/HomeCoordinator.swift

```diff
@@ -24,7 +24,10 @@ final class HomeCoordinator: NSObject,
         root.tabBarItem = UITabBarItem(title: GDSLocalisedString(stringLiteral: "app_homeTitle").value,
                                        image: UIImage(systemName: "house"),
                                        tag: 0)
-        let hc = HomeViewController(analyticsService: analyticsService)
+        let viewModel = ServicesTileViewModel.yourServices(analyticsService: analyticsService,
+                                                           urlOpener: UIApplication.shared)
```

Comment:
Is it worth considering defaulting the argument for `urlOpener` in the `yourServices` function to this? Eg:

```
func yourServices(analyticsService: AnalyticsService,
                  urlOpener: URLOpener = UIApplication.shared)
```

Reply:
Possibly.. I tried this & got a `Main actor-isolated class property 'shared' can not be referenced from a nonisolated context...` warning. I'm assuming this means I'd have to add a `@MainActor` annotation wherever its used?

Reply:
Yeah, would make sense for this to be added to the protocol in common.